etc/config/fsharp.amazon.properties

```diff
@@ -1,10 +1,10 @@
 compilers=&fsharp
-supportsBinary=false
+supportsBinary=true
 needsMulti=false
 compilerType=fsharp
 defaultCompiler=dotnet700fsharp
 
-group.fsharp.compilers=dotnet6011fsharp:dotnet700fsharp
+group.fsharp.compilers=dotnet6011fsharp:dotnet700fsharp:dotnettrunkfsharp
```

Comment:
we can adapt the versions order from gcc, clang and go in the ui: `dotnettrunkfsharp;dotnet700fsharp;dotnet6011fsharp` (same for c# and vb)

makes sense because someone experimenting on godbolt.org will likely want to test the latest version first (LTS version is not important to be the first when it comes to godbolt).

Reply:
Done.